Investigate/fix test_clicking_on_content_rating

At least on the marketplace.prod.* runs, test_clicking_on_content_rating is failing:

https://webqa-ci.mozilla.com/job/marketplace.prod.saucelabs/51/testReport/tests.desktop.consumer_pages.test_details_page/TestDetailsPage/test_clicking_on_content_rating/

Sauce Labs job: http://saucelabs.com/jobs/13b222ae0d4143918d27b142bba75379

Stack:

=================================== FAILURES ===================================
_______________ TestDetailsPage.test_clicking_on_content_rating ________________
[gw5] linux2 -- Python 2.7.9 /var/lib/jenkins/jobs/marketplace.prod.saucelabs/workspace/.env/bin/python2.7
tests/desktop/consumer_pages/test_details_page.py:123: in test_clicking_on_content_rating
>       home_page.set_region("restofworld")
pages/desktop/consumer_pages/base.py:64: in set_region
>       debug_page = self.go_to_debug_page()
pages/desktop/consumer_pages/base.py:60: in go_to_debug_page
>       return Debug(self.testsetup)
pages/desktop/regions/debug.py:18: in __init__
>       self.wait_for_element_visible(*self._region_select_locator)
pages/page.py:73: in wait_for_element_visible
>       while not self.is_element_visible(*locator):
pages/page.py:57: in is_element_visible
>           return self._selenium_root.find_element(*locator).is_displayed()
.env/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:664: in find_element
>                            {'using': by, 'value': value})['value']
.env/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py:173: in execute
>       response = self.command_executor.execute(driver_command, params)
.env/lib/python2.7/site-packages/selenium/webdriver/remote/remote_connection.py:349: in execute
>       return self._request(command_info[0], url, body=data)
.env/lib/python2.7/site-packages/selenium/webdriver/remote/remote_connection.py:417: in _request
>           resp = opener.open(request)
/usr/local/lib/python2.7/urllib2.py:431: in open
>       response = self._open(req, data)
/usr/local/lib/python2.7/urllib2.py:449: in _open
>                                 '_open', req)
/usr/local/lib/python2.7/urllib2.py:409: in _call_chain
>           result = func(*args)
/usr/local/lib/python2.7/urllib2.py:1227: in http_open
>       return self.do_open(httplib.HTTPConnection, req)
/usr/local/lib/python2.7/urllib2.py:1200: in do_open
>               r = h.getresponse(buffering=True)
/usr/local/lib/python2.7/httplib.py:1073: in getresponse
>       response.begin()
/usr/local/lib/python2.7/httplib.py:415: in begin
>           version, status, reason = self._read_status()
/usr/local/lib/python2.7/httplib.py:371: in _read_status
>       line = self.fp.readline(_MAXLINE + 1)
/usr/local/lib/python2.7/socket.py:476: in readline
>                   data = self._sock.recv(self._rbufsize)
E                   Failed: Timeout >400s
------------------------------- pytest-mozwebqa --------------------------------
Failing URL: https://marketplace.firefox.com/debug
------------------------------- Captured stderr --------------------------------

+++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++

~~~~~~~~~~~~~~~~~~~~~ Stack of receiver (140513575122688) ~~~~~~~~~~~~~~~~~~~~~~
  File "/usr/local/lib/python2.7/threading.py", line 783, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/local/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/local/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/var/lib/jenkins/jobs/marketplace.prod.saucelabs/workspace/.env/lib/python2.7/site-packages/execnet/gateway_base.py", line 674, in _thread_receiver
    msg = Message.from_io(io)
  File "/var/lib/jenkins/jobs/marketplace.prod.saucelabs/workspace/.env/lib/python2.7/site-packages/execnet/gateway_base.py", line 121, in from_io
    header = io.read(9) # type 1, channel 4, payload 4
  File "/var/lib/jenkins/jobs/marketplace.prod.saucelabs/workspace/.env/lib/python2.7/site-packages/execnet/gateway_base.py", line 91, in read
    data = self._read(numbytes-len(buf))

--- a/pages/desktop/regions/debug.py
+++ b/pages/desktop/regions/debug.py
@@ -11,7 +11,7 @@ from pages.page import Page
 
 class Debug(Page):
 
-    _region_select_locator = (By.ID, 'region')
+    _region_select_locator = (By.NAME, 'region')
 
     def __init__(self, testsetup):
         Page.__init__(self, testsetup)

